DigiMeSDK/Core/Classes/Entities/DMEServiceGroup.m

```diff
@@ -0,0 +1,59 @@
+//
+//  DMEServiceGroup.h
+//  DigiMeSDK
+//
+//  Created on 21/11/2019.
+//  Copyright © 2019 digi.me Limited. All rights reserved.
+//
+
+#import "DMEServiceGroup.h"
+
+static NSString * const kServiceGroupId = @"id";
```

Comment:
Can remove these static strings, along with `NSCoding` and `NSCopying` functions. This comment also applies to `DMEServiceType` and `DMEServiceObjectType`